### ISSUE TITLE
[Feature request] Pick what fields are displayed when voting

### ISSUE THREAD

**Issue by Emkai** [microsoft/azure-boards-estimate#154]
When voting we do not see all the field that contain valuable information. 
We have a field called Definition of Done that we would like to look at when voting.

**Comment by AminTi** [comment]
@Emkai  thanks, good suggestion.

**Comment by doormalena** [comment]
+1 We would like to see the area path

**Comment by S-OAltr** [comment]
+1 Our Team would like to view custom fields which are important to them as well

**Comment by Sebas3440** [comment]
+1 same here with our team

**Comment by Morten4** [comment]
+1 Same for us, can't see the fields that we use

**Comment by mirheegh** [comment]
+1 Same for me, it would be really handy to be customisable. My teams still use external tools because of this.

**Comment by ItsScroopz** [comment]
+1 My team could use this as well.

**Comment by carahnicole** [comment]
+1 here as well!

**Comment by kcotney** [comment]
Would be very helpful to show only exactly the fields needed in the game view to simplify information as much as possible when voting on a large number of items :)

**Comment by andrah** [comment]
Adding the "Steps to reproduce" area for the bugs would be really helpful.

**Comment by ptulou** [comment]
Yep, we got moved from a vanilla template (that worked great) to one that now has us splitting info for bugs across Description, Steps To Reproduce, and Acceptance Criteria. Only Description and Acceptance Criteria are being shown. It would be great if we could specify the "extra" field of Steps To Reproduce to show as well.

**Comment by khartung-fid** [comment]
+1 here as well!
